.github/ISSUE_TEMPLATE/sonarcloud_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Request for import GitHub repository on SonarCloud
+about: To import GitHub repository on SonarCloud.
+title: ''
+labels: sonarcloud-repo
+assignees: caggles, ShellyXueHan
+
+---
+
+## Step 0
+**Is your repository on `bcgov` GitHub Organization?**
+We have install SonarCloud on `bcgov` GitHub Organization for application teams to use. If your repo is not on bcgov, please migrate first.
+
+**Are you the admin user of the GitHub repositories?**
+The setup process require a repo admin level access. You will be assigned as the admin of your SonarCloud repos.
```

Comment:
Could make it a requirement to be the admin of your repo to make the request, and just close/cancel any requests where the requestor is NOT the repo admin?
eg:
```suggestion
The request requires a repository administrator to make the request.  The requestor will be assigned as the SonarCloud repository admin.
> Requests from non-Git repository admins will be closed/cancelled.
```

Reply:
yea that could be more clear!